Header header
duration t_track
duration t_corr
duration t_clones
float32 fps
geometry_msgs/Pose2D corr_offset
geometry_msgs/Pose2D ptw
float32 zncc
float32 RMS 
int16 stop
float32[9] H

